app/services/scheduler.py

```
@@ -178,110 +176,158 @@ def _add_stats_cleanup_job(self):
-        # Create a new event loop for this thread if needed
-        try:
-            loop = asyncio.get_event_loop()
-        except RuntimeError:
-            loop = asyncio.new_event_loop()
-            asyncio.set_event_loop(loop)
-
-        loop.run_until_complete(remote_transfer_service.process_pending_transfers())
-    except Exception:
-        logger.exception("Error in remote transfer job")
+        locations = db.query(ColdStorageLocation).all()
+        logger.info(f"Checking disk space for {len(locations)} cold storage locations")
```

Ruff (0.14.13)

Logging statement uses f-string

(G004)